parlai/core/dict.py

```diff
@@ -383,8 +383,6 @@ def tokenize(self, text, building=False):
 
     def bpe_tokenize(self, text):
         """Returns a sequence of BPE-tokens from the text."""
-        if self.lower:
```

Comment:
why remove?

Reply:
this wasn't really supposed to be there, this check is supposed to be in the tokenize function which will then call bpe_tokenize after lower'ing the text